packages/tasks/src/local-apps.ts

```diff
@@ -47,10 +47,11 @@ function isGgufModel(model: ModelData) {
 }
 
 const snippetLlamacpp = (model: ModelData): string => {
-	return `./main \
-	--hf-repo ${model.id} \
-	-m file.gguf \
-	-p "I believe the meaning of life is " -n 128`;
+	return `./main 
+	--hf-repo ${model.id} 
+	-m file.gguf 
+	-p "I believe the meaning of life is " 
+	-n 128`;
```

Comment:
ah yes the problem is that this will be displayed on newlines but won't be copy/pasteable in your terminal

I think maybe you need to escape them with two backslashes `\\`?